index.html

```diff
@@ -1,26 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<title>Homework Assignment Three</title>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
+		<div class="topbar clearfix">
+		<br>
+			<div class="leftfloat">Relaxr</div>
+			<div class="middlefloat"></div>
```

Comment:
This element (`<div class="middlefloat"></div>`) isn't actually doing anything so you can just remove it.